docs/4.2/admin-guide.md

```diff
@@ -172,6 +173,11 @@ of them is configurable.
 | `/usr/local/bin/tctl` | Teleport admin tool. It is only needed for auth servers.|
 | `/var/lib/teleport` | Teleport data directory. Nodes keep their keys and certificates there. Auth servers store the audit log and the cluster keys there, but the audit log storage can be further configured via `auth_service` section in the config file.|
 
+#### tsh
+| Full path                 | Purpose                                                                                                                                                                                                                                |
+|---------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `~/.tsh` | Stores temporary SSH certificates, stores user environment variables for reading when `permit_user_env: true`, and cache data such as proxy and user names from last logins. |
```

Comment:
@fspmarshall do you think this is a correct description for `~/.tsh`?

Reply:
I would just reword this to. 

> Temporary store for Proxy Address, Username,  SSH certificates and user environment variables when reading `permit_user_env: trues.

Reply:
revised